docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -2,20 +2,26 @@
 .. _module_dev_tutorial_sample:
 
 *******************************************
```

Comment:
I think all the current errors are caused by the two sets of heading markers here. Somehow instead of replacing the longer markers, you added a new pair of them . . .
```suggestion
```